cypress/integration/main-page.spec.js

```diff
@@ -1,3 +1,5 @@
+import { it } from 'mocha';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/bandvov/first-nextjs/pullRequest?prid=7125704)
```suggestion
import { it } from "mocha";
```